Lib/pdb.py

```diff
@@ -237,6 +240,44 @@ def namespace(self):
         )
 
 
+class _ZipTarget(_ExecutableTarget):
+    def __init__(self, target):
+        import runpy
+
+        self._target = os.path.realpath(target)
+        sys.path.insert(0, self._target)
+        try:
+            _, self._spec, self._code = runpy._get_main_module_details()
+        except ImportError as e:
+            print(f"ImportError: {e}")
```

Comment:
Why do you need to special-case ImportError here?

Reply:
That's from #108791. This is a very common case but the exception traceback would be very large because it's generated rather deeply and it distracts users. It would be a better experience to simply show that there's an import error when the user passes in an invalid module.

Reply:
Ok. there is also traceback.format_exception_only().

Reply:
Yeah I think the orignal thought here was just to suppress the common import error, and leave the others be. In that case we need a special case for `ImportError` anyway with or without `format_exception_only`.